index.bs

```diff
@@ -2136,7 +2154,8 @@ and a [=suitable origin=] |reportingOrigin|:
     : "<code>[=source debug data type/source-storage-limit=]</code>"
     : "<code>[=source debug data type/source-success=]</code>"
     : "<code>[=source debug data type/source-unknown-error=]</code>"
-    :: Return the result of running [=check if cookie-based debugging is allowed=] with |reportingOrigin|.
+    :: 1. If |debugCookieSet| is true, return <strong>allowed</strong>.
+    :: 1. Otherwise, return <strong>blocked</strong>.
```

Comment:
This shows up in the spec preview as two separate lists with ordinal `1` instead of `1, 2`.